consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/events/BlockTimerExpiry.java

```diff
@@ -34,8 +34,8 @@ public BlockTimerExpiry(final ConsensusRoundIdentifier roundIdentifier) {
   }
 
   @Override
-  public IbftEvents.Type getType() {
-    return IbftEvents.Type.BLOCK_TIMER_EXPIRY;
+  public BftEvents.Type getType() {
+    return BftEvents.Type.BLOCK_TIMER_EXPIRY;
   }
 
   public ConsensusRoundIdentifier getRoundIndentifier() {
```

Comment:
I know it's not part of your change. But would be nice to fix the misspelling here. /getRoundIndentifier/getRoundIdentifier